src/components/searchInput/types.ts

```diff
@@ -0,0 +1,72 @@
+import {TextInputProps, StyleProp, ViewStyle, TextStyle, ActivityIndicatorProps} from 'react-native';
+import {ButtonProps} from '../button';
+
+export enum SearchInputPresets {
+  DEFAULT = 'default',
+  PROMINENT = 'prominent'
+}
+
+export interface SearchInputRef {
+  blur: () => void;
+  clear: () => void;
+  focus: () => void;
+}
+
+export type SearchInputProps = TextInputProps & {
+  /**
+   * On clear button callback.
+   */
+  onClear?: () => void;
+  /**
+   * callback for dismiss action
+   */
+  onDismiss?: () => void;
+  /**
+   * Props for the cancel button
+   */
+  cancelButtonProps?: ButtonProps;
+  /**
+   * Custom right element
+   */
+  renderCustomRightElement?: (props?: any) => React.ReactElement;
+  /**
+   * Title prop
+   */
+  title?: string;
```

Comment:
Where do we use the title prop?

Reply:
I don't think we even use it in private...

Reply:
you are right, removed.